test/external/tqperf/CMakeLists.txt

```diff
@@ -15,6 +15,6 @@ nrn_add_test(
   CONFLICTS nmodl # https://github.com/BlueBrain/nmodl/issues/794
   REQUIRES mpi python coreneuron
   PROCESSORS ${tqperf_mpi_ranks}
-  NRNIVMODL_ARGS -loadflags -lcrypto
+  NRNIVMODL_ARGS -legacytransformations cxx -loadflags -lcrypto
```

Comment:
we should fix our tests (prior to merge?)

Reply:
Agree, but as in https://github.com/neuronsimulator/nrn/pull/1597#discussion_r832362132 I think the best approach depends on whether we're going to update these models to assume C++ or just to be compatible with it.